var/spack/repos/builtin/packages/llvm-amdgpu/package.py

```diff
@@ -228,6 +228,11 @@ def cmake_args(self):
             self.define("CLANG_ENABLE_AMDCLANG", "ON"),
         ]
 
+        if spec.target.family == "aarch64":
+            args.append(self.define("LLVM_TARGETS_TO_BUILD", "AMDGPU;AArch64"))
+        else:
+            args.append(self.define("LLVM_TARGETS_TO_BUILD", "AMDGPU;X86"))
```

Comment:
Can we use `get_llvm_targets_to_build` from the LLVM recipe?

Reply:
I feel like it might be better just to deal with this as a one-off for now, and leave any coupling to the larger llvm package for the future.